src/lib/public/board/data/index.ts

```diff
@@ -15,7 +17,7 @@ export const TEAMS = [...TEAMS_JSON].reduce(
     tt[t.id] = t;
     return tt;
   },
-  {}
+  {} as Record<keyof typeof TEAMS_JSON[number], Team>
```

Comment:
I think the type of `TEAMS` is still `{ [key: string]: Team; }` instead of `{ [key: keyof typeof TEAMS_JSON]: Team }`. 
```suggestion
  {}
```